sdk/go/common/util/cmdutil/console.go

```diff
@@ -152,7 +152,7 @@ func MeasureText(text string) int {
 	return uniseg.GraphemeClusterCount(clean)
 }
 
-func (table *Table) ToStringWithGap(columnGap string) string {
+func (table Table) ToStringWithGap(columnGap string) string {
```

Comment:
I'm curious why the pointer receiver was dropped here, versus added elsewhere?

Reply:
Lint requires the receivers to be of the same type, either all pointer types or all value types.

Reply:
Which linter are you using? I think it's different from our style guide and that rule causes issues with marshaling. See:

https://go.dev/play/p/w7fGZQBjmp6

For example, in `sdk/go/common/resource/config/key.go`, the tests fail unless `MarshalJJSON` is a value receiver and `UnmarshalJSON` is a pointer receiver.